api/models/user/user_model.py

```diff
@@ -59,10 +59,16 @@ class User(AbstractUser):
     """Custom Corona Testing  user implementation
     """
 
-    email = models.EmailField(unique=True)
-    username = models.CharField(max_length=25, null=True, blank=True)
-    age = models.IntegerField(null=True, blank=True)
+    # email = models.EmailField(unique=True)
+    
+    # age = models.IntegerField(null=True, blank=True)
+
+    username = models.CharField(max_length=25, null=True, blank=True, unique=False)
```

Comment:
Why are we keeping the username field if we use the phonenumber hash now?

Reply:
For setting unique=False. Otherwise throwing a duplicate error

Reply:
Otherwise it throws a duplicate username error

Reply:
Weird, I'll investigate